backend/amd64/emit.mlp

```diff
@@ -41,7 +41,9 @@ let _label s = D.label ~typ:QWORD s
 
 (* Override proc.ml *)
 
-let simd_frontend_disabled () = not (Language_extension.is_enabled SIMD)
+let assert_simd () =
```

Comment:
Nice! Could this go into `Arch` to avoid another copy of it in `proc.ml` ?
```suggestion
let assert_simd_extension_enabled () =
```

Reply:
Sure, moved + renamed